kani-driver/src/harness_runner.rs

```diff
@@ -116,7 +116,10 @@ impl KaniSession {
         let failing = failures.len();
         let total = succeeding + failing;
 
-        if self.args.concrete_playback.is_some() && !self.args.quiet && failures.is_empty() {
+        if self.args.concrete_playback.is_some()
+            && !self.args.quiet
+            && results.iter().all(|r| !r.result.generated_concrete_test)
+        {
             println!(
                 "INFO: The concrete playback feature never generated unit tests because there were no failing harnesses."
```

Comment:
Do you think we even need this? I was also wondering if we print that when a unit tests wasn't generated because it already existed.

Reply:
> I was also wondering if we print that when a unit tests wasn't generated because it already existed.

We do that [here](https://github.com/model-checking/kani/blob/f635f7ba833d532ca31d4eec95ff2339e2289869/kani-driver/src/concrete_playback.rs#L98).